cmd/user-ssh-keys-agent/Makefile

```
@@ -13,12 +14,46 @@
 # limitations under the License.
 
 DOCKER_REPO ?= "quay.io/kubermatic"
-GOOS ?= $(shell go env GOOS)
+IMAGE = "user-ssh-keys-agent"
+ARCHITECTURES ?= amd64 arm64
 
-.PHONY: build
-build:
-	GOOS=$(GOOS) CGO_ENABLED=0 go build -o ./_build/user-ssh-keys-agent
+.ONESHELL:
 
 .PHONY: docker
-docker: build
-	docker build -t $(DOCKER_REPO)/user-ssh-keys-agent:$(TAG) .
+docker:
+	cd ../.. ; \
+	docker build -t $(DOCKER_REPO)/user-ssh-keys-agent:$(TAG) -f cmd/user-ssh-keys-agent/Dockerfile .
+
+.PHONY: build
+build: build_multiarch
+	MANIFEST=${DOCKER_REPO}/${IMAGE}:${TAG} ; \
+	buildah manifest create "$$MANIFEST" ; \
+	for ARCH in ${ARCHITECTURES} ; do \
+		buildah manifest add "$$MANIFEST" "${DOCKER_REPO}/${IMAGE}-$$ARCH:${TAG}" ; \
+	done
+
+.PHONY: push
+push:
+	MANIFEST=${DOCKER_REPO}/${IMAGE}:${TAG} ; \
+	buildah manifest push --all "$$MANIFEST" "docker://$$MANIFEST"
+
+.PHONY: tag
+tag:
+	OLD_MANIFEST=${DOCKER_REPO}/${IMAGE}:${PRIMARY_TAG} ; \
+	NEW_MANIFEST=${DOCKER_REPO}/${IMAGE}:${NEW_TAG} ; \
+	buildah tag $$OLD_MANIFEST $$NEW_MANIFEST 
```

I think this needs no double $ (escape of $), isn't it?
```suggestion
	buildah tag $OLD_MANIFEST $NEW_MANIFEST 
```

---

Yeah, they are needed. Make requires double $$ to access rule-scoped variables